# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/InterceptedHttpChannel.cpp
# Commit: 02933fde7e3f
# Full Hash: 02933fde7e3f3cf4827730cb670dc76b91022813
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2020-10-13 21:43:51
# Regressor Bug: 1665062
# File Overlap Count: 1
# Description:
#   Bug 1665062: HTTPS-Only: Upgraded website creating HTTP auth prompt gets interrupted by error-page r=necko-reviewers,dragana,JulianWels
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91908
# ==============================================================================

diff -r bc531943d78a -r 02933fde7e3f netwerk/protocol/http/InterceptedHttpChannel.cpp
--- a/netwerk/protocol/http/InterceptedHttpChannel.cpp	Tue Oct 13 18:07:04 2020 +0300
+++ b/netwerk/protocol/http/InterceptedHttpChannel.cpp	Tue Oct 13 14:06:28 2020 +0000
@@ -553,6 +553,11 @@
 }
 
 NS_IMETHODIMP
+InterceptedHttpChannel::GetIsAuthChannel(bool* aIsAuthChannel) {
+  return NS_ERROR_NOT_IMPLEMENTED;
+}
+
+NS_IMETHODIMP
 InterceptedHttpChannel::SetPriority(int32_t aPriority) {
   mPriority = clamped<int32_t>(aPriority, INT16_MIN, INT16_MAX);
   return NS_OK;